sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -214,6 +214,13 @@ def setUp(self):
       url = self.fs.join(self.tmpdir, filename)
       self.fs.create(url).close()
 
+  def assertItemsEqual(self, l1, l2):
```

Comment:
I think we should switch to assertCountEqual, see also: https://issues.apache.org/jira/browse/BEAM-5620

Reply:
sg if we decide to use  six.assertCountEqual.  I am however confused by the comments in that thread, which on the other hand says "we've removed the dependency on six module". 

Do we have a consensus which route to take?

Reply:
We don't want to use six.

Reply:
Pinged that thread, let's repeat https://github.com/apache/beam/blob/c34c367f5da6f9bef8a46471195470923a201af9/sdks/python/apache_beam/coders/coders_test_common.py#L61

Reply:
sg.